src/less/window-notice/base/window-notice.less

```diff
@@ -50,3 +50,15 @@
         width: 100%;
     }
 }
+
+[dir="rtl"] {
+    .window-notice__header {
+        display: flex;
```

Comment:
We shouldn't really need to modify the display type in the RTL block. These rules may need moving to the base styles instead.

Reply:
I deleted the `display: flex` there and turns out it wasn't needed.